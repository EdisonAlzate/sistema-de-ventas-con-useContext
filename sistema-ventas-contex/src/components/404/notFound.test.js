import { render } from '@testing-library/react';
import NotFound from './notFound';

test('Renders correctly page notfound', () => {
    const { container } = render(<NotFound />);
    expect(container).toMatchSnapshot();
});
