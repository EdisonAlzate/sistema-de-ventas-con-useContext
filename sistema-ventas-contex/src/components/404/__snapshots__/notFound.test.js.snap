// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly page notfound 1`] = `
<div>
  <div
    style="display: flex; justify-content: center; align-items: center;"
  >
    <h1>
      404! Pagina no encontrada
    </h1>
  </div>
</div>
`;
